test/export/InstanceExporter.test.ts

```diff
@@ -6876,7 +6907,7 @@ describe('InstanceExporter', () => {
 
       const exported = exportInstance(carePlanInstance);
       expect(exported.instantiatesCanonical).toEqual(undefined); // instantiatesCanonical is not set with invalid type
-      expect(loggerSpy.getMessageAtIndex(1, 'error')).toMatch(
+      expect(loggerSpy.getMessageAtIndex(7, 'error')).toMatch(
```

Comment:
What's causing the six additional errors here?

Reply:
I believe the extra error messages were due to how this test was calling exportInstance multiple times (exportInstance is defined as a function in the beforeEach that when called, calls export for cs, vs, and sd and the instance given) so on the second time, it logged the errors. I adjusted this test (and the one above) to just call export within the function instead.